lib/Textile/Models/index.ts

```diff
@@ -1,5 +1,9 @@
 import { AppStateStatus } from 'react-native'
 
+export interface BufferJSON {
```

Comment:
I really wanted to keep this file for models that need to match models defined in Go. It makes it easier to keep them in sync. I'm not sure if you've already added a bunch of other stuff to this file or not, but if possible, it would be good to put SDK-specfic models in their own file.

Reply:
makes sense, will split them out